node/bft/src/worker.rs

```diff
@@ -623,7 +623,7 @@ mod tests {
         let ledger: Arc<dyn LedgerService<CurrentNetwork>> = Arc::new(mock_ledger);
 
         // Ensure the maximum number of redundant requests is correct and consistent across iterations.
-        assert_eq!(max_redundant_requests(ledger, 0), 34, "Update me if the formula changes");
+        assert_eq!(max_redundant_requests(ledger, 0), 6, "Update me if the formula changes");
```

Comment:
Hmm is it possible to test the `34` value as well? By using a higher round or height? Or does the test ledger not simulate having higher block heights?

Same for other similar tests.

Reply:
> Or does the test ledger not simulate having higher block heights?

That's precisely the issue; our tests usually pick a fresh ledger and start at round 0.

Reply:
Not sure if this is going too deep into the rabbithole, but maybe for at least one test, you can see if with `mock_ledger.expect_*` you can return a larger committee.

Reply:
Noting for posteriority: we could also adjust the function signature of `max_redundant_requests` to take in `num_validators` to test different aspects.

Given how simple the function is, let's defer.